src/org/omegat/core/data/RealProject.java

```diff
@@ -279,6 +281,7 @@ public void createProject() {
             createDirectory(config.getDictRoot(), OConsts.DEFAULT_DICT);
             createDirectory(config.getTargetRoot(), OConsts.DEFAULT_TARGET);
             //createDirectory(m_config.getTMOtherLangRoot(), OConsts.DEFAULT_OTHERLANG);
+            GlossaryManager.createNewWritableGlossaryFile(config.getWritableGlossaryFile());
```

Comment:
This is completely new, right? Before we were only creating the writable glossary file on-demand, but now we are doing it proactively on project creation.

Is there a reason to do it here instead of on-demand?

Reply:
Oh, I do it for test. I'd like to remove.

Reply:
But it may be better to create on project creation because now we put a help comment.
Reported bug may happen when user create a file,  but it is not able to detect file encoding.

Reply:
We create the file on-demand when appending via the Create Glossary Entry dialog. I changed it so that it creates the file with your new method, so it will have the magic comment.